ui/app/components/app-footer.hbs

```diff
@@ -0,0 +1,31 @@
+{{!
+  Copyright (c) HashiCorp, Inc.
+  SPDX-License-Identifier: BUSL-1.1
+~}}
+
+<Hds::AppFooter as |AF|>
+  <AF.ExtraBefore>
+    {{#if this.isDevelopment}}
+      <div class="env-banner">
+        <div class="level-item notification">
+          <Icon @name="git-branch" /><Icon @name="pencil-tool" />
+          Local development
+          <Icon @name="pencil-tool" /><Icon @name="git-branch" />
+        </div>
+      </div>
+    {{/if}}
+  </AF.ExtraBefore>
+  <AF.Link @href={{changelog-url-for this.version.version}} data-test-footer-version>
+    Vault
+    {{this.version.version}}
+  </AF.Link>
+  {{#if (is-version "OSS")}}
```

Comment:
I'm confused about how the `is-version` helper plays with the new community/version service updates. 

Maybe we should pull in the version service to the component and use that instead?

Reply:
Yeah - upon further inspection it looks like this conditional should be updated because I'm still seeing the upgrade prompt even though I'm on enterprise
<img width="775" alt="Screenshot 2023-11-30 at 4 07 27 PM" src="https://github.com/hashicorp/vault/assets/68122737/2826eda2-26a3-4275-9b8b-aef3ed4359f2">

Reply:
That cleanup could be out of the scope for this work, alternately this could just be updated to `{{#unless (is-version "Enterprise")}}`, or resolved any number of ways! I'll defer to you

Reply:
is-version still pulls from the `isEnterprise` and `isOSS` values from the version service, so I didn't need to update it. I'll look into why it isn't working in that case though, and might end up swapping it for `isEnterprise` directly